webapp/views/launch/script.js

```diff
@@ -410,30 +410,33 @@ function handleResponse(res) {
         const columns = list.map(col => `<option value="${col}">${col}</option>`);
         const rel = ['AND', 'OR', 'NOT'].map(el => `<option value="${el}">${el}</option>`);
         const op = ['>', '<', '=', '>=', '<='].map(el => `<option value="${el}">${el}</option>`);
+        const firstCondition = `
+        <div class='d-flex'>
+          <select class='${messageId}-input custom-select mr-2'>${columns}</select>
+          <select class='${messageId}-input custom-select mr-2'>${op}</select>
+          <input class='${messageId}-input form-control' type="number" />
+        </div>
```

Comment:
You could use jQuery more

Reply:
fixed

Reply:
I see no changes here

Reply:
> I see no changes here

Sorry I missed this last time. I will change this in the next pull request